tests/integration/advanced/test_custom_modules.py

```diff
@@ -0,0 +1,52 @@
+from lightwood.api.high_level import json_ai_from_problem, code_from_json_ai, predictor_from_code
+from lightwood.api.types import JsonAI, ProblemDefinition
+import unittest
+from mindsdb_datasources import FileDS
+import os
+import shutil
+
+
+test_err_message = 'This ! Is ! A ! Testing ! Error !'
+
+
+def create_custom_module():
+    mdir = os.path.expanduser('~/lightwood_modules')
+
+    try:
+        shutil.rmtree(mdir)
```

Comment:
Wouldn't this test remove all custom modules a user might have added?

Reply:
Yeap, we should probably only remove the file we create,, good point.

Reply:
Might be worth considering a "refresh modules" by removing anything custom function. Either way, I like it.

Dumb question, how do users import custom accuracy functions as in if they want to do ```from sklearn.metrics import hinge_loss``` with this new approach?

Reply:
Any accuracy function gets auto-imported:
`accuracy_function = getattr(importlib.import_module('sklearn.metrics'), accuracy_function_str)`